ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/ontologies/views/AppCategorizeViewImpl.java

```diff
@@ -42,12 +43,14 @@
     @UiField(provided = true) OntologiesView.OntologiesViewAppearance appearance = GWT.create(OntologiesView.OntologiesViewAppearance.class);
 
     final private Widget widget;
+    private OntologyUtil ontologyUtil;
 
     @Inject
     public AppCategorizeViewImpl(final TreeStore<OntologyHierarchy> treeStore) {
         this.treeStore = treeStore;
         initCategoryTree();
         widget = uiBinder.createAndBindUi(this);
+        ontologyUtil = GWT.create(OntologyUtil.class);
```

Comment:
Why not just OntologyUtil.getInstance()? Just curious because I have not see this usage before.